subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -148,15 +148,10 @@ You'll find that the new Gradle build includes the following:
 
  * All the custom repositories that are specified in the POM
  * Your external and inter-project dependencies
- * The appropriate plugins to build the project (limited to one or more of the <<maven_plugin#maven_plugin,Maven>>, <<java_plugin#java_plugin,Java>> and <<war_plugin#war_plugin,War>> Plugins)
+ * The appropriate plugins to build the project (limited to one or more of the <<publishing_maven,maven-publish>>, <<java_plugin,java>> and <<war_plugin,war>> plugins)
```

Comment:
As above, I'm trying to standardise on plugin names rather than plugin IDs. So "Maven Publish, Java and War Plugins".

Reply:
Done